pkg/discovery/discovery.go

```diff
@@ -30,14 +30,21 @@ import (
 	"github.com/rifflock/lfshook"
 	"github.com/sirupsen/logrus"
 	"github.com/viniciuschiele/tarx"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/kubernetes"
 
 	pluginaggregation "github.com/heptio/sonobuoy/pkg/plugin/aggregation"
 )
 
-// Run is the main entrypoint for discovery
+// Deprecated during 0.14.0; should be removed by 0.16.0. Previously the main entrypoint
```

Comment:
@stevesloka thoughts on deprecation vs removal?

These are exported methods so people could consume them in code, but if thats the case they can also pin the version they want/need.

These are not intrinsically part of the sonobuoy client interface and is really just consumed by the `sonobuoy master` command itself.

Reply:
Why are you deprecating? If they aren't used anywhere else I'd just throw away and remove the confusion.